### DISCUSSION TITLE
Updating developer roles

### DISCUSSION THREAD

**Discussion by kmanimaran** [gruntwork-io/knowledge-base#570] [Help]
How can one update roles for developer after making changes as outlined in https://github.com/gruntwork-io/terraform-aws-security/tree/main/modules/iam-policies? 

In which folder should 'terragrunt apply' be run to update the roles? Running it under security baseline folder doesn't seem to update the role. 

@iangrunt 

---

<ins datetime="2022-10-07T16:41:32Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109336">Tracked in ticket #109336</a></p>
</ins>

**Comment by josh-padnick** [comment]
The trick here is that you'll need to find which Terraform module is creating the actual IAM roles. You can see this in action in [this example](https://github.com/gruntwork-io/terraform-aws-security/blob/main/examples/iam-users/main.tf#L115), where one module is used to create the IAM Users, and then in the same module the `iam-policies` module is used.

It sounds a bit hacky, but one way to do that is to merely search the git repo for an occurrence of a string like `source = "https://github.com/gruntwork-io/terraform-aws-security/iam-policies"`. Note that your actual code reference will look a little different depending on how you're accessing the repo.
